bne.lic

```diff
@@ -67,10 +85,67 @@ class BNE
         end
       end
     end
+
     bput('burgle', 'You make short work of the lock', 'You scale up the side of a wall')
-      #for each room_list  - tbd
-    fput('go window')
+
+    #stow entry method if needed once inside - easier loot managment
+    stow_hands
+
+    #TODO:  Define list of room types, and searchable objects to search for loot (https://elanthipedia.play.net/Breaking_and_Entering#Rooms)
+    #       Write movement and get out routines
+    if @bne_settings['search_count'] > 0
+      search_for_loot('counter')
+    end
+
+    bput('go window', 'You take a moment to reflect on the caper')
+
+  end
+
+  def search_for_loot(target)
+    if /It looks valuable/ =~ bput("search #{target}",'It looks valuable','Roundtime','I could not')
+      if @bne_settings['dispose']
+        drop_item("#{DRC.right_hand}")
```

Comment:
💡 The string interpolation isn't necessary here (or line 109)

Reply:
I think I just left that in from when I was using bput and didn't fully clear it when I ripped the code out of steal.  I can fix that with the pausing other scripts too.